assets/dev/js/frontend/handlers/image-carousel.js

```diff
@@ -1,4 +1,6 @@
-export default class ImageCarousel extends elementorModules.frontend.handlers.CarouselBase {
+import CarouselBase from 'elementor-frontend/handlers/base-carousel';
```

Comment:
@louiswol94 The type of this import isn't so important, as we don't extend the ImageCarousel class inside the Pro plugin. But we can use this approach for consistency.